pkg/minikube/logs/logs.go

```diff
@@ -188,12 +189,27 @@ func Output(r cruntime.Manager, bs bootstrapper.Bootstrapper, cfg config.Cluster
 		}
 	}
 
+	outputAudit(lines, failed)
+
 	if len(failed) > 0 {
 		return fmt.Errorf("unable to fetch logs for: %s", strings.Join(failed, ", "))
 	}
 	return nil
 }
 
+// outputAudit displays the audit logs.
+func outputAudit(lines int, failed []string) {
+	out.Step(style.Empty, "")
+	out.Step(style.Empty, "==> Audit <==")
+	a, err := audit.Retrieve(lines)
```

Comment:
I think this should return an object that has the report, and that "Report" object has a method for table

for example

r:=. audit.Report(lines)

r.Table()

and simmiliarly later we can have different formats

r.Json()

r.Markdown()

Reply:
Good idea, changed it to follow this format